scripts/build-windows-provisioning-scripts.sh

```diff
@@ -32,11 +33,12 @@ fi
 
 CREATE_TIME="$(date +%s)"
 az login --service-principal -u $CLIENT_ID -p $CLIENT_SECRET --tenant $TENANT_ID
+az group create -n $RESOURCE_GROUP_NAME -l $STORAGE_LOCATION
```

Comment:
Can you rebase your changes?
build-windows-provisioing-scripts.sh in master has LOCATION added and the same changes as here.

Reply:
https://github.com/Azure/aks-engine/blob/master/scripts/build-windows-provisioning-scripts.sh

Reply:
waiting until #4012 is merged